client/index.html

```diff
@@ -45,6 +47,24 @@
                     mockKeyType: keyType,
                 });
             }
+
+            function signVestingPayout(toOwner = true) {
+                keyguard.signTransaction({
+                    type: TransactionType.EXTENDED,
+                    sender: 'NQ74 FLQL DRE3 99PF CET0 3N7D JKLF MQP6 87KS',
+                    senderLabel: 'Vesting Contract',
+                    senderType: 1, // Vesting
+                    recipient: toOwner ? 'NQ47 FS55 KNXG 25XL 37N8 LD78 1DDH 8CS0 QBNF' : 'NQ94 VESA PKTA 9YQ0 XKGC HVH0 Q9DF VSFU STSP',
+                    signer: 'NQ47 FS55 KNXG 25XL 37N8 LD78 1DDH 8CS0 QBNF',
+                    signerLabel: 'Vesting Owner',
+                    value: 100,
+                    fee: 0.00138,
+                    network: 'test',
+                    validityStartHeight: 100000,
+                    extraData: 'Finally, I can move my stash!',
+                    mockKeyType: EncryptionType.HIGH,
+                });
+            }
         </script>
 </body>
 </html>
```

Comment:
Can't this entire file be moved to demos? It's even titled "KeyguardClient Demo" ;)

Reply:
Will be removed in a separate PR.